entrypoint.sh

```diff
@@ -1,3 +1 @@
-#!/bin/bash
-
-poetry run gunicorn --bind "0.0.0.0:${PORT:-5000}" "todo_app.app:create_app()"
\ No newline at end of file
+kubectl port-forward service/module-14 7080:80
```

Comment:
What was the intention here?

This entrypoint.sh file is executed by your container. Your todo app container has no idea what `kubectl` is. In fact it looks like you released this to Azure and now your app is broken. [See logs here](https://portal.azure.com/#@softwireacademy.onmicrosoft.com/resource/subscriptions/d33b95c7-af3c-4247-9661-aa96d47fccc0/resourceGroups/McLaren1_GeorgeAnker_ProjectExercise/providers/Microsoft.Web/sites/ganker-todo-terraformed-as/logStream)

Your container still needs to run your application. Kubernetes is just a way of running the container (and scaling with copies of it, forwarding requests, etc). `kubectl` is a command line tool that exists in your machine, not inside the todo app image.